spec/ADAS/ADAS.vspec

```diff
@@ -83,10 +83,14 @@
   type: sensor
   description: Indicates if obstacle sensor system incurred an error condition. True = Error. False = No Error.
 
-- ObstacleDetection.DistanceToObject:
+- ObstacleDetection.Detector:
```

Comment:
I am fine with "Detector", I have nit checked the whole ADAS structure, but is "ObstacleDetection.Detection not double? So either have branches

Obstacle.Detector. xxxx

or

ADAS.ObstacleDetector -> INsances
(but not sure about isError and isActive... But lets say side cam broken, you could still see forward....

Reply:
Use Obstacle instead of sensor?

Reply:
@danielwilms wrote:
> Use Obstacle instead of sensor?

I don't understand.  Where do you see "sensor" in the current example?  Is it in comparison to Sebastian's proposal or your own (which uses "Detector")?

I think "Obstacle" is not a synonym for "Sensor", so what more generally are we trying to achieve with the change?
Maybe I'm missing something.

@SebastianSchildt wrote:
>but is "ObstacleDetection.Detection not double? 

I don't see ObstacleDetection.**Detection**, I see ObstacleDetection.**Detector** ?
Sorry to nitpick, but it matters for understanding of the conversation.
(IMHO, Detection+Detector is not duplication in itself.  The only thing that matters then is if we really want to model the "detector" and if that is the best word (compared to for example sensor).

Overall, you might know I'm not a fan of adding too much depth to the tree if not needed.  It is the data that matters to me, not to model the actual physical setup of the car.   So, a "Sensor" node should only be mentioned if we need the extra level to reasonably explain the meaning of the data items.  

Often "<Concept>.<DataItem>" is fine for me.  It is therein _implicit_ and well known that there is some sensor that will be measuring the DataItem, but that's not necessary in the data taxonomy.   We are not _primarily_ modeling a physical car, we are primarily  organizing data items, in my opinion.   (In many cases we also do not know or care where and how the data item was actually measured).

So as an example `ObstacleDetection.DistanceToObject` was fine for me for a single object and I am likely to prefer it over `ObstacleDetection.MeasuringThing.DistanceToObject` no matter if you call MeasuringThing, "Detector" or "Sensor" or something else.

But I think there might be more to do here -  is a single MeasuringThing (sensor) instance able to detect more than one object for example?  If it is, then putting DistanceToObject as a child of a sensor-instance still does not help, because there are still multiple instances of the distance.

I think it might be easier to just use this PR for what it is for (remove instances) and avoid renaming anything.  And then discussing renaming separately?